components/family-studies/ui/src/main/resources/PhenoTips/FamilyRecordField_externalId.xml

```diff
@@ -112,20 +112,11 @@
 
 {{velocity}}
 #if($xcontext.action == 'edit' || $doc.display('external_id') != '')
-  #if ($xcontext.action != 'edit')
-    (((#__form_label('external_id')
-       #__form_field('external_id'))))
-  #else
-    (% class="fieldset external_id" %)(((
-      (% class="half-width external_id" %)(((
-        #__form_label('external_id' 'section')
-      )))
-      (% class="half-width external_id" %)(((
-      #__form_field('external_id')
-      )))
-      (% class="clear" %)((()))
-    )))
-  #end
+  (% class="fieldset external_id" %)(((
+    #__form_label('external_id' 'section')
+
```

Comment:
Are you sure this blank line is needed? In view mode, it causes the layout to be inconsistent, the value is on a separate line.